### ISSUE TITLE
SOPS breaks when a config file is found, but no rules match the current file

### ISSUE THREAD

**Issue by slikk66** [mozilla/sops#611]
````
$ cat .env 
TEST=123

$ sops -e .env 
error loading config: no matching creation rules found

$ sops -e --input-type dotenv .env 
error loading config: no matching creation rules found

$ sops -e --input-type dotenv --output-type dotenv .env 
error loading config: no matching creation rules found

$ sops -e --input-type=dotenv --output-type=dotenv .env 
error loading config: no matching creation rules found

$ sops -e --input-type=dotenv --output-type=dotenv --kms arn:aws:kms:us-east-1:111111111111:key/222222222-3333-4444-5555-666666666666 .env
error loading config: no matching creation rules found

$ sops --version
sops 3.5.0 (latest)
````

Am I doing something wrong?
Thanks!

**Comment by autrilla** [comment]
Looks like you have a .sops.yaml config file, but none of the rules match your file name (.env).

I guess you found a bug :) sops should proceed as if there was no config file in that case.

**Comment by slikk66** [comment]
thanks, I just realized that there was indeed a .sops.yaml file higher up in the tree, when I removed it I was able to encrypt

Thanks!

**Comment by Kamahl19** [comment]
@autrilla Same thing happens when I only want to use `.sops.yaml` to define destination_rule for Vault publishing. However I dont want to specify creation rules. Could it be considered a bug as well?

**Comment by autrilla** [comment]
It's kind of the same bug, yeah.

**Comment by Kamahl19** [comment]
@autrilla What solution would you suggest? I need to get this solved so I am open to making a PR, but it would be super helpful if you could hint me the solution. This would be my first go contribution :)

**Comment by autrilla** [comment]
Fixed in #662
